docs/rum.asciidoc

```diff
@@ -2,51 +2,43 @@
 = Real User Monitoring (RUM)
 [partintro]
 --
-The {apm-rum-ref}/index.html[JavaScript RUM Agent] is supporting Real User Monitoring.
-You can make use of <<sourcemaps, source maps>> when using Elastic RUM. 
+Real User Monitoring captures user interaction with clients such as web browsers.
+The {apm-rum-ref}/index.html[JavaScript Agent] is Elastic's RUM Agent.
+To use it you need to <<rum-enable,enable RUM support>> in the APM Server.
 --
 
 [[sourcemaps]]
 == Source Maps
-It is common practice to minify JavaScript code for several reasons, e.g. performance gain. 
-This can make debugging very difficult, as it is hard to read the minified files.
-Source mapping can help for debugging minfied JavaScript files, 
-by mapping code from the minified files to the original source code. 
+It is common practice to minify JavaScript code, e.g. to reduce network latency.
```

Comment:
again I would avoid using e.g. I don't have a strong preference personally, but it's been drilled into me by editors over the years. :-)

Reply:
ah, i guess this answers my comment https://github.com/elastic/apm-server/pull/1169#discussion_r205080708
:)

ill change it then 👍